style.css

```diff
@@ -0,0 +1,220 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    max-width: 1024px;
+    margin: 0 auto;
+    font-family: 'opensans-regular', sans-serif;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 30px;
+    color: #838C95;
+    background: #161415;
+}
+
+header {
+    position: relative;
+    height: 600px;
+    min-height: 500px;
+    background: #161415;
+    background-size: cover;
+    overflow: hidden;
+}
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+
+.social {
+    margin: 24px 0;
+    padding: 0;
+    font-size: 30px;
+    text-shadow: 0px 1px 2px rgba(0, 0, 0, .8);
+    display: inline-block;
+    text-align: center;
+}
+
+.social li {
+    display: inline-block;
+    margin: 0;
+    padding: 0;
+    margin-left: 42px;
+}
+
+a {
+    color: #11ABB0;
+    text-decoration: none;
+}
+
+nav {
+    background-color: #333;
+    font: 12px 'Arial', sans-serif;
+    text-transform: uppercase;
+    letter-spacing: 2.5px;
+    width: 100%;
+    margin: 0 auto;
+    z-index: 100;
+    position: fixed;
+    left: 0;
+    top: 0;
+}
+
+ul#nav {
+    min-height: 48px;
+    width: auto;
+    text-align: center;
+}
+
+ul#nav li {
+    position: relative;
+    list-style: none;
+    height: 48px;
+    display: inline-block;
+}
+
+ul#nav li a {
+    display: inline-block;
+    padding: 8px 13px;
+    line-height: 32px;
+    text-decoration: none;
+    text-align: left;
+    color: #fff;
+    transition: color .2s ease-in-out;
+}
+
+.row {
+    width: auto;
+}
+
+section {
+    background: #fff;
+    padding-top: 96px;
+    padding-bottom: 66px;
+    overflow: hidden;
+}
+
+.row {
```

Comment:
in line 86 you set already some property to .row, put all styles in one place, please.